rollup.config.js

```diff
@@ -1,26 +1,58 @@
+/* tslint:disable */
 // PLUGINS
 import typescript from 'rollup-plugin-typescript';
 import commonjs from 'rollup-plugin-commonjs';
 
+import * as packageJSON from './package.json';
+
 export default {
-  input: './src/index.ts',
-  output: [
-    { file: 'commonjs/index.js', format: 'cjs', sourcemap: true },
-    { file: 'esm/index.es.js', format: 'es', sourcemap: true }
-  ],
-  plugins: [
-    typescript({ tsconfig: './tsconfig.json' }),
-    commonjs({
-      include: 'node_modules/**'
-    })
-  ],
-  external: [
-    'react',
-    'styled-components',
-    '@rebass/grid',
-    'classnames',
-    'lodash',
-    'rxjs',
-    'polished'
-  ]
+    input: {
+        index: './src/index.ts',
+        autocomplete: './src/AutoComplete/AutoComplete.component.tsx',
```

Comment:
90% sure these should all point to the index.ts files, but I wanted to play with this some.